src/Conversion/ONNXToTOSA/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# set(LLVM_TARGET_DEFINITIONS ONNXToTOSALegalizePatterns.td)
```

Comment:
It's always better to not check in commented out code. I think this should all be removed.

Reply:
Oops, this was a missing git add post cleanup. Fixed.